### PR TITLE
Grow 274

### PR DESCRIPTION
**Pull request by BoulderBrains**

### CODE REVIEW

**Comment by BoulderBrains** [comment]
Changed route and reworked components after the discussion with design last week. Page frame now looks like this: 
Screen Shot 2020-10-19 at 3 08 05 PM

File: src/router/routes.js [hunk 1]
```diff
@@ -184,6 +184,19 @@ export default [
 		],
 
 	},
+	{
+		path: '/settings/security',
+		component: () => import('@/pages/Settings/SecuritySettings'),
+		children: [{
+			path: '',
+			name: 'password',
```
Comment (eddieferrer) [inline src/router/routes.js:192]: Are we sure about the name on this route? We don't typically have names on routes...

File: src/router/routes.js [hunk 2]
```diff
@@ -184,6 +184,19 @@ export default [
...
+			component: () => import('@/pages/Settings/SecuritySettingsPage'),
+			meta: {
+				activeLoginRequired: true,
+			}
+		},
+		]
+	},
 	{ path: '/settings/security/mfa', component: () => import('@/pages/Settings/TwoStepVerifyPage') },
```
Comment (eddieferrer) [inline src/router/routes.js:200]: You can make this a child of the settings/security route above

Comment (BoulderBrains) [reply src/router/routes.js:200]: The TwoStepVerifyPage has it's own header and footer and putting this as a child of settings/security results in: 
Screen Shot 2020-10-19 at 3 50 10 PM

File: src/router/routes.js (hunk 1 above)
Comment (BoulderBrains) [reply src/router/routes.js]: removed, thanks for calling it out

**Comment by emuvente** [comment]
I think SecuritySettings.vue should be combined with SecuritySettingsPage.vue into a single component that doesn't use a router-view component. I think that will allow the 2-step verification page to be used as that child route without causing problems 🤔

File: src/router/routes.js (hunk 2 above)
Comment (eddieferrer) [reply src/router/routes.js:199]: Yeah you're right, I'm fine with leaving it as is then

File: src/pages/Settings/SecuritySettings.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+export default {
+	components: {
+		WwwPage,
+	},
+};
+</script>
+
+<style lang="scss">
```
Comment (ryan-ludwig) [inline src/pages/Settings/SecuritySettings.vue:24]: Please make sure the styles stay "scoped"

Comment (ryan-ludwig) [reply src/pages/Settings/SecuritySettings.vue:24]: On the other pages as well

**Comment by BoulderBrains** [comment]
@ryan-ludwig I've scoped the css of both files
@emuvente
- I've combine SecuritySettings & SecuritySettingsPage into one component that doesn't use the <router-view>
- In order for both routes to load with the correct components, I had to use two different routes, each with a paths and component.

**Comment by BoulderBrains** [comment]
I would like to get this merged before branch cut today, any other thoughts suggestions? @emuvente @eddieferrer @ryan-ludwig

File: src/pages/Settings/Password.vue [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<template>
+	<div class="row">
+		<div class="column small-12 large-8">
```
Comment (ryan-ludwig) [inline src/pages/Settings/Password.vue:3]: I believe this class should be "columns"

File: src/pages/Settings/TwoStepVerification.vue [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<template>
+	<div class="row">
+		<div class="column small-12 large-8">
```
Comment (ryan-ludwig) [inline src/pages/Settings/TwoStepVerification.vue:3]: same "columns"

**Review by ryan-ludwig — APPROVED** [review]
Minor class name thing, but otherwise looks good!

**Review by eddieferrer — APPROVED** [review]

File: src/pages/Settings/Password.vue (hunk 4 above)
Comment (eddieferrer) [reply src/pages/Settings/Password.vue:3]: You can use either column or columns, since there is only 1, in this case 'column' makes grammatical sense

**Pull request merged**